.storybook/preview.tsx

```diff
@@ -32,6 +32,10 @@ const globalStyles = css`
     color: var(--echoes-color-text-default);
     background-color: var(--echoes-color-background-default);
   }
+
+  p {
+    margin: 0;
+  }
```

Comment:
I think we shouldn't have that. If we expect the Display component to reset paragraphs margins then it should do so in it's implem, we can't expect the products to have this reset.

Reply:
As discussed, it doesn't handle margins at all